src/navigation.tpl

```diff
@@ -1,16 +1,10 @@
  <ul>
     <li><a href="about-juju.html">About Juju</a></li>
-    <li><a href="introducing-2.html">Introducing Juju 2.0!</a></li>
-    <li><a href="whats-new.html">What's new in 2.1</a></li>
 <!-- SECTION -->
-    <li class="section"><h4 class="header toggle-target">Get Started</h4>
+    <li class="section"><h4 class="header toggle-target">Quick Guides</h4>
         <ul>
-        <li class="section"><a class="header" href="getting-started.html">First use!</a>
-            <ul class="sub">
-                <li><a href="getting-started-general.html">Install for non-16.04</a></li>
-            </ul>
-        </li>
-        <li><a href="getting-started-jaas.html">Juju as a Service (JAAS)</a></li>
+        <li class="section"><a class="header" href="getting-started.html">First use!</a></li>
+        <li><a href="tut-lxd.html">Local Juju with LXD cloud</a></li>
```

Comment:
I think we can come up with a better name for this. For one thing, LXD isn't the benefit, so it is rather irrelevant to messaging (LXD is how we deliver what they want).
Alternative suggestions - Juju on a no-cost, local cloud